circle.yml

```diff
@@ -1,36 +1,33 @@
-machine:
-  java:
-    version: oraclejdk8
-  environment:
-    TERM: dumb
```

Comment:
it tells Gradle to use a naive output mode rather than the normal terminal pretty-print node.